docs/plugins_es.md

```diff
@@ -0,0 +1,373 @@
+---
+title: Plugins
+order: 700
+lang: es
+---
+
+# Plugins
+
+Scully utiliza un sistema de plugins que permite a los usuarios definir nuevas formas en las que Scully haga el pre-render de la aplicación. Hay cinco tipos 
+principales de plugins:
+
+1. [Router Plugin](#router-plugin)
+2. [Render Plugin](#render-plugin)
+3. [File Handler Plugin](#file-handler-plugin)
+4. [routeDiscoveryDone plugin](#routediscoverydone-plugin)
+5. [allDone plugin](#alldone-plugin)
+
+Puede encontrar una lista de plugins disponibles la documentación de [plugins recomendados ](recommended-plugins_es.md).
+
+---
+
+## Cómo Registerar un Plugin
+
+La función `registerPlugin` agrega un nuevo plugin a Scully y tiene 5 parámetros:
+
+- type
+- name
+- plugin
+- validator (optional)
+- options (optional)
+
+### type: string
+
+`type` - Indica el tipo de the plugin. Los tipos existentes son: `router`, `render`, `fileHandler`, `allDone`, o `routeDiscoveryDone`.
+
+### name: string
+
+`name` - El nombre del plugin. Este debe ser único para el tipo de plugin. Para remplazar un plugin existente, agregue la opción `replaceExistingPlugin`.
+
+### plugin: any
+
+`plugin` - La función del plugin. Los tipos de plugins estan explicados en su propia descripción.
+
+### validator: función (optional)
+
+`validator` - Función de validación que debe regresar un arreglo de errores. Si no hay errores, debe regresar un valor `false`. Si regresa un `string<array>`, esos strings son los erroresa mostrar y el proceso en ejecición es abortado.
+
+> Tip: Agregar color a los errores de validació utilizando la extensión de color de Scully.
+
+##### Ejemplo de un Validator
+
+```typescript
+import { yellow } from '@scullyio/scully';
+
+// Código omitido
+
+const validator = async options => {
+  const errors = [];
+
+  if (options.numberOfPages && typeof options.numberOfPages !== 'number') {
+    errors.push(
+      `my-custom-plugin numberOfPages should be a number, not a ${yellow(
+        typeof options.numberOfPages
+      )}`
+    );
+  }
+
+  return errors;
+};
+```
+
+### Opciones
+
+El objeto `optinal` puede ser utilizado para agregar las opciones del plugin. Por el momento, la única opcioón disponible es `replaceExistingPlugin`.
```

Comment:
```markdown
El objeto `optinal` puede ser utilizado para agregar las opciones del plugin. Por el momento, la única opcioón disponible es `replaceExistingPlugin`.
```

No me queda muy claro cual fue la idea acá, pero hay varios typos. 
- `optinal`
- `opcioón`

```markdown
El objeto  `options` el cual es opcional,  puede ser utilizado para agregar las opciones del plugin. Por el momento, la única opción disponible es `replaceExistingPlugin`.
```

Reply:
I rewrote this, so it is more clear. I also fixed the typo.